tests/backup/backup_longevity_test.go

```diff
@@ -0,0 +1,452 @@
+package tests
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/portworx/torpedo/pkg/log"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	. "github.com/portworx/torpedo/tests"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	testTriggersConfigMap = "longevity-triggers"
```

Comment:
Should we have different config map for backup related triggers?

Reply:
Sure, will change this to pxb-longevity-triggers